{
  "contest": 229,
  "handle": "Breeje",
  "risk": "2",
  "title": "Funds can be stuck in FeeWrapper contract when call() fails silently",
  "issueId": 1159,
  "issueUrl": "https://github.com/code-423n4/2023-04-rubicon-findings/issues/1159"
}